tests/helpers/test_storage.py

```diff
@@ -1,6 +1,7 @@
 """Tests for the storage helper."""
 import asyncio
 from datetime import timedelta
+from ipaddress import IPv4Address
 import json
 from unittest.mock import Mock, patch
```

Comment:
Import `patch` from `asynctest` instead. It's backwards compatible. We need to use that until Python 3.8 is our minimum.

Reply:
Aha, thank you :) I was just going to use new_callable=MagicMock but this sounds better.

Reply:
Hmm, that throws a bunch of deprecated warnings in 3.8. Is that acceptable?

Reply:
I think so yes, at least for now.